.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: Build macOS, Ubuntu, Windows
+
+on:
+  # TEMPORARY change used to test achubbic
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+*'
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            profile: release+static
+            target: x86_64-unknown-linux-musl
+            needs_musl: true
```

Comment:
```suggestion
#         - os: ubuntu-latest
#           profile: release+static
#           target: x86_64-unknown-linux-musl
#           needs_musl: true
```

maybe comment this out so we dont have to run an extra build for now until we decide to move jenkins but like @silverjam said, it's negligible, (kinda like having two duplicate tests running same time)